appstore-metadata-service-tests/src/test/groovy/com/lgi/appstore/metadata/test/cases/real/sanity/DevApiFTSpecSanity.groovy

```diff
@@ -173,6 +177,7 @@ class DevApiFTSpecSanity extends AsmsSanitySpecBase {
         field().header().version().from(theBody1) == v1
         field().header().visible().from(theBody1) == v1Visible
         field().header().encryption().from(theBody1) == v1Encryption
+        field().header().preferred().from(theBody1) == v1Preferred
```

Comment:
if there's no test for retrieval app by preferred field, would be great to add one

Reply:
Added